visqol-rs/src/lib.rs

```diff
@@ -51,7 +51,7 @@ mod similarity_to_quality_mapper;
 mod spectrogram;
 mod spectrogram_builder;
 mod speech_similarity_to_quality_mapper;
-mod support_vector_regression_model;
+pub mod support_vector_regression_model;  /// make public
```

Comment:
```suggestion
pub mod support_vector_regression_model;
```
Comments like this belong IMO in the commit message.